typescript.js

```diff
@@ -4,6 +4,9 @@ module.exports = {
   rules: {
     "typescript/indent": "off",
     "typescript/member-delimiter-style": "off",
-    "typescript/type-annotation-spacing": "off"
+    "typescript/type-annotation-spacing": "off",
```

Comment:
Yes, because the project `typescript-eslint-plugin` is archived now.
Then you need to bump up major version because the remove a API, isn't it.
Since you might want to keep API for backward version, I couldn't add these changes.